numpy/core/numeric.py

```diff
@@ -2185,9 +2185,9 @@ def allclose(a, b, rtol=1.e-5, atol=1.e-8, equal_nan=False):
     ----------
     a, b : array_like
         Input arrays to compare.
-    rtol : float
+    rtol : array_like
```

Comment:
Just a quick comment: if `rtol` and `atol` can be arrays, it should also be possible to dispatch on them, so one needs to change `_isclose_dispatcher` and `_allclose_dispatcher` as well.

Reply:
Thanks, I wasn't familiar with that machinery before. Done - LMK if that looks right!